tests/integration/actions/run/test_stdout_tmux.py

```diff
@@ -79,6 +79,15 @@ class ShellCommand(UiTestStep):
         ).join(),
         present=["usage: ansible-playbook [-h]"],
     ),
+    ShellCommand(
```

Comment:
The test should include a playbook that prompts, this might be a little tricky since I don't think we have an example of one. I'm happy to help with this